addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -71,8 +71,9 @@ var SnippetEditor = Widget.extend({
      * @param {Object} templateOptions
      * @param {jQuery} $editable
      * @param {Object} options
+     * @param {Boolean} [allowParentsEditors=true]
      */
-    init: function (parent, target, templateOptions, $editable, options) {
+    init: function (parent, target, templateOptions, $editable, options, allowParentsEditors = true) {
```

Comment:
Not a fan of adding a parameter (even optional) in stable (but 17.4 would be ok) but really not a fan of adding a parameter after an "option" one. Instead of this: make `allowParentsEditors` part of the `options` param.